.travis.yml

```diff
@@ -1,23 +1,29 @@
 language: python
 sudo: false
-python:
-  - 2.6
-  - 2.7
-  - 3.2
-  - 3.3
-  - 3.4
-  - 3.5
-  - nightly
-  - pypy
-  - pypy3
 install:
-  - pip install -e .
+  - pip install tox
   - pip list
-script:
-  - python pycodestyle.py --testsuite testsuite
-  - python pycodestyle.py --statistics pycodestyle.py
-  - python pycodestyle.py --doctest
-  - python setup.py test
+script: tox
```

Comment:
I like this part! I had come up with the same idea and forgotten to change it... Great minds! :)